spec/integration/normalizer_spec.rb

```diff
@@ -1,20 +1,10 @@
 require 'rails_helper'
 
 RSpec.describe 'Normalizer' do
-  def create_raw_event
-    create :raw_event, data: File.read("#{Rails.root}/spec/fixtures/request_2.json"),
-                       application: application
-  end
-
-  let(:application) { create :application }
-  let(:raw_event) { create_raw_event }
-
-  before do
-    worker = ParseRawEventJob.new
-    worker.perform raw_event.id
-  end
+  let(:record_with_sql) { { sql: "select * from users where channel_id in (1, 2, 3, 4)" } }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.